spec/units/commands/factory_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require 'gitsh/commands/factory'
+
+describe Gitsh::Commands::Factory do
+  describe '#build' do
+    it 'returns an instance of the given command class' do
```

Comment:
@hrs and I were both unhappy with this spec, but we weren't sure how to improve it.